src/components/highlight-component/highlight-component.js

```diff
@@ -44,9 +44,9 @@ function HighlightComponent () {
 
     // the method will add a click event (listener), it'll then open a new window with the documentationUrl for that component.
     component.element.addEventListener('click', function (event) {
-      event.stopPropagation() // prevent event bubbling
-      event.preventDefault()
       if (this.isComponentsHighlighted) {
+        event.stopPropagation() // prevent event bubbling
+        event.preventDefault()
```

Comment:
OMG thanks for fixing this! It's been [on our backlog](https://gov-uk.atlassian.net/browse/WHIT-1924) but never prioritised, partly because we thought it might be a big job. Interesting to see how few changes were needed!